pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Monitor TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Monitor TCP sockets from Loopback interfaces:
+
+`nettop -m {{tcp}} -t {{loopback}}`
+
+- Monitor a specific process:
+
+`nettop -p {{process_id|process_name}}`
+
+- Display summary only for per-process:
+
+`nettop -P`
+
+- Monitor in logging mode and display 10 samples:
+
+`nettop -l {{10}}`
+
+- Monitor in delta mode, updating every 5 seconds:
+
+`nettop -d -s {{5}}`
+
+- List interactive commands while running nettop:
```

Comment:
```suggestion
- While running nettop, list interactive commands:
```

Reply:
OK, it's clearer than before, I will change it.

Reply:
Oh, I just found out that GitHub can commit suggestions directly, it's so cool. Looks like I'm still lacking in knowledge about GitHub.